sdk/internal/uuid/uuid.go

```diff
@@ -6,45 +6,28 @@
 package uuid
 
 import (
+	"crypto/rand"
 	"fmt"
-	"math/rand"
 	"strconv"
-	"time"
 )
 
 // The UUID reserved variants.
 const (
 	reservedRFC4122 byte = 0x40
 )
 
-func init() {
-	rand.Seed(time.Now().Unix())
-}
-
 // A UUID representation compliant with specification in RFC 4122 document.
 type UUID [16]byte
 
 // New returns a new uuid using RFC 4122 algorithm.
 func New() UUID {
 	u := UUID{}
-	// Set all bits to randomly (or pseudo-randomly) chosen values.
-	// math/rand.Read() is no-fail so we omit any error checking.
+	// Set all bits to pseudo-random values.
 	// NOTE: this takes a process-wide lock
```

Comment:
Do we know if the 'process-wide lock' bit is still true?

Reply:
According to the [docs](https://pkg.go.dev/math/rand#Read), there is no mention of a process wide lock, I think this note can be removed in this case.

Reply:
It's not in the docs, you have to look at the impl.  For `crypto/rand` the comment might not be applicable.

Reply:
For the Unix impl it also takes a lock.